wrims-gui/build.gradle

```diff
@@ -0,0 +1,56 @@
+plugins {
+    id 'library.deps-conventions'
+    id 'library.java-conventions'
+    id "de.undercouch.download" version "5.6.0"
+}
+
+def wrimsGuiFolder = "wrims2_gui_x64_${version}"
+def wrimsGuiDownload = "${buildDir}/download/wrims2_gui_download.zip"
+
+//download the latest WRIMS GUI installer from DWR
+tasks.register('downloadWrimsGuiInstaller', Download) {
+    src 'https://data.cnra.ca.gov/dataset/0f6b03b4-7de8-4579-8aa0-60f73d9d21fb/resource/89f60e4a-c0c2-4ca2-acac-beb6bdb5d69e/download/wrims2_gui_x64_20240129.zip'
```

Comment:
Was this change in requirements to download the latest installer instead of tracking and deploying template coordinated? If so, I still see a template in the GitHub packages and a publish-wrims-template gradle subproject, which should get cleaned up.

Reply:
Josh is deleting